modules/mono/glue/cs_files/Vector2.cs

```diff
@@ -220,13 +220,13 @@ public Vector2 Tangent()
         }
         
         private static readonly Vector2 zero   = new Vector2 (0, 0);
-	    private static readonly Vector2 one    = new Vector2 (1, 1);
```

Comment:
Could you squash that in the original commit?

Reply:
I've never done that before. After looking it up, I am confused, don't you have the option to "squash and merge"?

Reply:
We do, but it has various flaws, so we don't use it. See http://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#mastering-the-pr-workflow-the-rebase

Reply:
Thanks for that article link, it's quite helpful. I was able to figure it out, it's now just one commit.